setup.py

```diff
@@ -216,6 +216,9 @@
           'test': test_requires,
       },
       dependency_links=[
+          'https://github.com/ip-tools/mongodb_gridfs_beaker/archive/0.6.0dev1.tar.gz#egg=mongodb_gridfs_beaker',
+          'https://github.com/ip-tools/mechanize/archive/v0.4.3dev2.tar.gz#egg=mechanize-0.4.3dev2',
+          #'https://github.com/dagwieers/unoconv/archive/master.tar.gz#egg=unoconv-0.8.2',
       ],
```

Comment:
In order to be able to run a general availability release, and publish the package to PyPI, we will need to resolve those dependencies beforehand.

Reply:
Can it be solved by including submodules in the source path?

Reply:
We could vendorize the dependencies, but we still will need to port them over to Python 3. I think there is not so much work to do. On `mongodb_gridfs_beaker`, I made a start at [^1], which is probably already pulled in here. On `mechanize`, a minor patch was required, but it's already upstream, see [^2].

[^1]: https://github.com/loucash/mongodb_gridfs_beaker/pull/1
[^2]: https://github.com/python-mechanize/mechanize/pull/43